src/main/rx/rx_spi.c

```diff
@@ -116,6 +118,20 @@ STATIC_UNIT_TESTED bool rxSpiSetProtocol(rx_spi_protocol_e protocol)
         protocolSetRcDataFromPayload = frSkyDSetRcData;
         break;
 #endif
+#ifdef USE_RX_FLYSKY
```

Comment:
Since both of `USE_RX_FLYSKY` and `USE_RX_FLYSKY_A2` use the same hardware, they should really be combined into one driver, and the driver should choose the right sub-protocol based on `rxConfig->rx_spi_protocol` that is passed into the `protocolInit` function.